### ISSUE TITLE
Get followers for a user

### ISSUE THREAD

**Issue by narodnik** [mahrtayyab/tweety#6]
Thanks for this great library.

I went through the user API and there is nothing to get the followers for a user. Is this possible? I noticed you cannot access followers without logging in.

**Comment by mahrtayyab** [comment]
> Thanks for this great library.
> 
> I went through the user API and there is nothing to get the followers for a user. Is this possible? I noticed you cannot access followers without logging in.

Yes , getting followers without logging in isn't possible , I am also working on a Client too ,  where you can log in and scrape more

**Comment by narodnik** [comment]
Thanks so much for your work
